src/main/java/core/basesyntax/Ellipse.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Ellipse extends Figure {
+    private double lengthA;
+    private double lengthB;
+
+    public Ellipse(double sideA, double lengthB) {
+        this.lengthA = sideA;
+        this.lengthB = lengthB;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * lengthA * lengthB;
+    }
+
+    @Override
+    public String printInformation() {
+        return "Figure: Rectangle, area: " + getArea()

```

Comment:
The `printInformation` method incorrectly refers to the figure as a 'Rectangle'. It should be 'Ellipse' to accurately describe the figure type.